src/dataValidate.js

```diff
@@ -0,0 +1,22 @@
+const { errorMessages } = require('./errorMessages');
```

Comment:
```suggestion
const { errorMessages } = require('./errorsMessage');
```

Or rename file as you write here